### ISSUE TITLE
CRITICAL ERROR: There are no account state for CRS

### ISSUE THREAD

**Issue by sondreb** [block-core/blockcore-wallet#265]
![image](https://user-images.githubusercontent.com/309938/196305600-f18f89e3-46e7-4021-81bc-ef313bf42ede.png)

**Comment by dangershony** [comment]
Do you know the fix?

**Comment by sondreb** [comment]
No, it happens in the indexer (local wallet indexer) if there are no account state available. It likely happened with quick wallet and not normal.

The only logical answer without more heavy debugging, is that the indexer process calls save on the store, right after user has added a new account. Then you can get into race condition.

![image](https://user-images.githubusercontent.com/309938/196422004-26d52041-9016-430d-b8b7-fc547c5d85a4.png)

**Comment by sondreb** [comment]
Got an identity account that worked yesterday, today it stopped working:

![image](https://user-images.githubusercontent.com/309938/196703522-06f89471-c3a8-4d8c-8063-47cb67eb6d24.png)

This might be some crash that happens while I'm developing, but it might also be an issue that needs further investigation.

**Comment by sondreb** [comment]
Had an existing STRAX wallet, attempted to add CRS account to the same wallet, and it failed:

![image](https://user-images.githubusercontent.com/309938/200043350-040a04ea-8608-45b6-88a0-a1bb215bf778.png)
